data-structure/linkedList.ts

```diff
@@ -0,0 +1,167 @@
+// eslint-disable-next-line max-classes-per-file,@typescript-eslint/ban-ts-comment
+// @ts-ignore
+// eslint-disable-next-line max-classes-per-file
+type Item = string | number;
+
+interface NodeObject {
+  value: Item;
+  next: object | null;
```

Comment:
'object' type not indicative enough. and you define 'next' by another type in class. it will confuse